config/checkstyle-checks.xml

```diff
@@ -46,6 +46,18 @@
     <property name="files"
               value=".*[\\/]src[\\/](test|it|xdocs-examples)[\\/].*(?&lt;!Support)\.java"/>
   </module>
+  <!--
+    Until we fully support opening-closing paragraph tag
+    https://github.com/checkstyle/checkstyle/issues/15685
+   -->
+  <module name="SuppressionSingleFilter">
+    <property name="checks" value="JavadocParagraph"/>
+    <property name="message" value="tag should be preceded with an empty line."/>
+  </module>
+  <module name="SuppressionSingleFilter">
+    <property name="checks" value="JavadocParagraph"/>
+    <property name="message" value="Redundant .* tag."/>
+  </module>
```

Comment:
suppression until we fix the implementation of this check. May change the referenced issue in future if we open new issue for it

Reply:
We need to fix almost all violations from Paragraph Check. https://github.com/checkstyle/checkstyle/blob/aeaf4b0838d814ce62ef23aa6b4d367ff5f83fb2/src/main/java/com/puppycrawl/tools/checkstyle/JavaAstVisitor.java#L84-L86 is good example. Please start a process of updating our sources, let's make PRs to be limited to ~20  files. All PRs will have "supplemental: " prefix.

Reply:
If there is some strange cases, report to us and skip them for now, we will handle them at the end of as soon as you get confirmation on how to update javadoc.

Reply:
we need fixes, please continue https://github.com/checkstyle/checkstyle/pull/15737 and all other fixes.
We eat food we produce before we release to it to users, we need to fix all of them.

Reply:
Put this link in comment , you will update all in separate PR

Reply:
this will be addressed at https://github.com/checkstyle/checkstyle/issues/15762